zio-http/shared/src/main/scala/zio/http/ServerSentEvent.scala

```diff
@@ -42,12 +46,12 @@ final case class ServerSentEvent[T](
   data: T,
   eventType: Option[String] = None,
   id: Option[String] = None,
-  retry: Option[Int] = None,
+  retry: Option[Duration] = None,
```

Comment:
Please add a constructor, that takes `Option[Int]` for compatibility.
Also adjust the docs of the param here.

Reply:
Added it, though, I realised I missed an edge case, when a negative int value is passed. So, this constructor ends up silently filtering it out as well.

Reply:
Please add back the constructor with the Int, but without default values.

Reply:
Done